src/ORM/EagerLoader.php

```diff
@@ -239,9 +239,7 @@ public function setMatching($assoc, callable $builder = null, $options = [])
 
         $pointer[$last] = ['queryBuilder' => $builder, 'matching' => true] + $options;
 
-        $this->_matching->contain($containments);
-
-        return $this;
+        return $this->_matching->contain($containments);
```

Comment:
I thought most setters are intended to return `$this` so they can be chained. Should we perhaps add more method calls in the call sites?